tests/unit/test_package_metadata.py

```diff
@@ -2,16 +2,17 @@
 
 import sys
 
+from packaging.version import Version
 from phylum import PKG_NAME, PKG_SUMMARY, __author__, __email__, __version__
 
 from ..constants import PYPROJECT
 
 
 def test_project_version():
-    """Ensure the source version matches the PEP-440 post-normalization format given to the package."""
-    package_version = __version__
-    source_version = PYPROJECT.get("tool", {}).get("poetry", {}).get("version")
-    assert package_version == source_version, "Source version should match PEP-440 post-normalization format"
+    """Ensure the source version can be normalized to the PEP-440 post-normalization format given to the package."""
+    package_version = Version(__version__)
+    source_version = Version(PYPROJECT.get("tool", {}).get("poetry", {}).get("version"))
+    assert package_version == source_version, "Source version unable to normalize to PEP-440 post-normalization format"
```

Comment:
If the version does not conform to PEP 440, the `Version(...)` constructor will raise an exception and we'll never get to this `assert`.

I still think the `assert` is valid, but I'm not actually sure what would make it fail. So I think it makes most sense to just leave the message blank rather than guessing.

```suggestion
    assert package_version == source_version
```